src/components/CodeAssignmentModal/index.jsx

```diff
@@ -97,6 +99,19 @@ export class BaseCodeAssignmentModal extends React.Component {
     this.setState({ mode });
   }
 
+  getCleanedUsers(emails, usersResponse) {
+    const users = [];
+    emails.forEach((email) => {
+      const user = usersResponse.find(userResponse => userResponse.email === email);
+      users.push({
+        email,
+        lms_user_id: user ? user.id : undefined,
```

Comment:
we don't want to send these keys if the value is not there.

Reply:
Key will not be sent if it is undefined.